roles/wait-for-convergence/tasks/main.yml

```diff
@@ -19,7 +19,7 @@
 - debug: msg="running wait_for_agent_convergence.sh {{ctrl_ip}} {{host_id.stdout.strip()}} {{du_username}} ******** <{{flags}}>"
 
 - name: wait for pf9-hostagent to converge
-  script: "files/wait_for_agent_convergence.sh {{ctrl_ip}} {{host_id.stdout.strip()}} {{du_username}} \'{{du_password}}\' {{flags}}"
```

Comment:
why was this removed? This reintroduces a bug that was fixed by adding it.

Reply:
weird, this breaks on my mac, maybe because of my python version?

Reply:
Not sure, we haven't tested this on Mac. We only support it on CentOS 7 and Ubuntu 16.04/18.04 today.